### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by dweindl**

### CODE REVIEW

File: examples/conversion_reaction/mainConversionReaction.m [hunk 1]
```diff
@@ -24,8 +26,8 @@
 % 
 % Profile likelihood calculation is done using getParameterProfiles().
 %
-% Multi-chain Monte-Carlo sampling is performed by getParameterSamples() 
-% and plotted using plotParameterUncertainty().
+% Sngle-chain Monte-Carlo sampling is performed by getParameterSamples()
```
Comment (paulstapor) [inline examples/conversion_reaction/mainConversionReaction.m]: Typo: Single

File: examples/enzymatic_catalysis/mainEnzymaticCatalysis.m [hunk 2]
```diff
@@ -26,10 +27,9 @@
 % optimization based on these measurements, demonstrating the use of
 % getMultiStarts().
 %
-% The Profile likelihoods are calculated in two different ways: First, the
-% calculation is done by repeated reoptimization using 
-% getParameterProfiles(), then it is done by integrating the an ODE along
-% the profile path using integrateParameterProfiles().
+% The Profile likelihoods are calculated by integrating the an ODE along
```
Comment (paulstapor) [inline examples/enzymatic_catalysis/mainEnzymaticCatalysis.m]: ...integrating _(the)_ an ODE...

File: examples/enzymatic_catalysis/mainEnzymaticCatalysis.m [hunk 3]
```diff
@@ -154,23 +154,9 @@
...
-    'MaxStep', 0.1, ...
-    'MinStep', 1e-5, ...
-    'MaxNumSteps', 1e4, ...
-    'RelTol', 1e-5, ...
-    'AbsTol', 1e-8, ...
-    'hessianStep', 1e-7, ...
-    'gradientStep', 1e-7 ...
-    );
+optionsPesto.solver.gamma = 10;
+optionsPesto.solver.type = 'ode113';
```
Comment (paulstapor) [inline examples/enzymatic_catalysis/mainEnzymaticCatalysis.m]: This line can be deleted

**Comment by dweindl** [comment]
Ok. @paulstapor

**Comment by paulstapor** [comment]
done.

**Pull request merged**